docs/usage/types/urls.md

```diff
@@ -2,56 +2,50 @@
-  - `postgresql+pg8000`
-  - `postgresql+psycopg`
-  - `postgresql+psycopg2`
-  - `postgresql+psycopg2cffi`
-  - `postgresql+py-postgresql`
-  - `postgresql+pygresql`
-- `MySQLDsn`: scheme `mysql`, user info required, TLD not required, host required. Also, its supported DBAPI dialects:
-  - `mysql`
-  - `mysql+mysqlconnector`
-  - `mysql+aiomysql`
-  - `mysql+asyncmy`
-  - `mysql+mysqldb`
-  - `mysql+pymysql`
-  - `mysql+cymysql`
-  - `mysql+pyodbc`
-- `MariaDBDsn`: scheme `mariadb`, user info required, TLD not required, host required. Also, its supported DBAPI dialects:
-  - `mariadb`
-  - `mariadb+mariadbconnector`
-  - `mariadb+pymysql`
-- `CockroachDsn`: scheme `cockroachdb`, user info required, TLD not required, host required. Also, its supported DBAPI dialects:
-  - `cockroachdb+asyncpg`
-  - `cockroachdb+psycopg2`
-- `AmqpDsn`: schema `amqp` or `amqps`, user info not required, TLD not required, host not required
-- `RedisDsn`: scheme `redis` or `rediss`, user info not required, tld not required, host not required (CHANGED: user info) (e.g., `rediss://:pass@localhost`)
-- `MongoDsn` : scheme `mongodb`, user info not required, database name not required, port
-  not required from **v1.6** onwards), user info may be passed without user part (e.g., `mongodb://mongodb0.example.com:27017`)
-- `stricturl`: method with the following keyword arguments:
-    - `strip_whitespace: bool = True`
-    - `min_length: int = 1`
-    - `max_length: int = 2 ** 16`
-    - `tld_required: bool = True`
-    - `host_required: bool = True`
-    - `allowed_schemes: Optional[Set[str]] = None`
-
-!!! warning
-    In V1.10.0 and v1.10.1 `stricturl` also took an optional `quote_plus` argument and URL components were percent
-    encoded in some cases. This feature was removed in v1.10.2, see
-    [#4470](https://github.com/pydantic/pydantic/pull/4470) for explanation and more details.
-
-The above types (which all inherit from `AnyUrl`) will attempt to give descriptive errors when invalid URLs are
-provided:
+- [`AnyUrl`][pydantic.networks.AnyUrl]: any scheme allowed, top-level domain (TLD) not required, host required.
+- [`AnyHttpUrl`][pydantic.networks.AnyHttpUrl]: scheme `http` or `https`, TLD not required, host required.
+- [`HttpUrl`][pydantic.networks.HttpUrl]: scheme `http` or `https`, TLD required, host required, max length 2083.
+- [`FileUrl`][pydantic.networks.FileUrl]: scheme `file`, host not required.
+- [`PostgresDsn`][pydantic.networks.PostgresDsn]: user info required, TLD not required, host required.
+    `PostgresDsn` supports multiple hosts. The following schemes are supported:
+    - `postgres`
+    - `postgresql`
+    - `postgresql+asyncpg`
+    - `postgresql+pg8000`
+    - `postgresql+psycopg`
+    - `postgresql+psycopg2`
+    - `postgresql+psycopg2cffi`
+    - `postgresql+py-postgresql`
+    - `postgresql+pygresql`
+- [`MySQLDsn`][pydantic.networks.MySQLDsn]: scheme `mysql`, user info required, TLD not required, host required.
```

Comment:
I wonder if we want to link to definitions of all of these, where possible? I assume it's easy enough for users to search for these though having links to e.g. mysql docs may be a nice touch? I suppose it's also a lot of effort to collect and maintain external links, so maybe not worthwhile.